src/components/signup.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState, useEffect } from "react";
+import { Link, useNavigate } from "react-router-dom";
+import { useDispatch, useSelector } from "react-redux";
+import sendSignupDetails from "../store/user_reducer";
+import "./SignUp.css";
+
+const Register = () => {
+  const navigate = useNavigate();
+  const homePage = () => {
+    navigate("/"), { replace: true };
+  };
+
+  const dispatch = useDispatch();
+  const state = useSelector((state) => state.user);
+  console.log(state);
```

Comment:
- [ ] Kindly delete this old piece of debugging console.log code, to improve the cleanliness of the repo. 👍